### ISSUE TITLE
Other API's not part of Supertokens API being blocked in Safari browser

### ISSUE THREAD

**Issue by Alengeorge01** [supertokens/supertokens-website#168]
Facing this issue where all backend API's including Supertokens API are not being triggered when initializing Supertokens in the constructor in the Safari Browser. I'm getting this error.

![image](https://user-images.githubusercontent.com/36100871/197173113-26160967-6e7d-4687-8b90-11dbef30eca9.png)

When removing `superTokens.init()` all the API calls are being made properly. It's working for other browsers.  Even the top-level domains for both Frontend and backend are the same.

Platforms used :

Frontend: Angular
Backend: Node JS

NPM Packages used in Frontend : `"supertokens-web-js": "^0.3.0"`

Code for Supertokens initialization:

```
SuperTokens.init({
        appInfo: {
          apiDomain: "https://blocksurvey-supertoken-function-m8v6.onrender.com",
          apiBasePath: "/auth",
          appName: "blocksurvey",
        },
        recipeList: [
          Session.init(),
          Passwordless.init()
        ],
      });
```

Below I have attached the stack trace as well:

```
[Error] ERROR – Error: Uncaught (in promise): TypeError: Illegal constructor
XMLHttpRequestProgressEvent@[native code]
L@https://np.blocksurvey.io/3898.8806d9d9cf0c97c3.js:1:1367947
@https://np.blocksurvey.io/3898.8806d9d9cf0c97c3.js:1:1372522
onInvoke@https://np.blocksurvey.io/main.8e46cddcf277f36d.js:1:347046
run@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:61614
@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:76769
onInvokeTask@https://np.blocksurvey.io/main.8e46cddcf277f36d.js:1:346866
runTask@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:62264
ie@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:69080
invokeTask@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:68088
Ee@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:80880
ve@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:81319
Error: Uncaught (in promise): TypeError: Illegal constructor
XMLHttpRequestProgressEvent@[native code]
L@https://np.blocksurvey.io/3898.8806d9d9cf0c97c3.js:1:1367947
@https://np.blocksurvey.io/3898.8806d9d9cf0c97c3.js:1:1372522
onInvoke@https://np.blocksurvey.io/main.8e46cddcf277f36d.js:1:347046
run@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:61614
@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:76769
onInvokeTask@https://np.blocksurvey.io/main.8e46cddcf277f36d.js:1:346866
runTask@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:62264
ie@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:69080
invokeTask@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:68088
Ee@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:80880
ve@https://np.blocksurvey.io/polyfills.813ff8f4ba583420.js:1:81319
	oy (main.8e46cddcf277f36d.js:1:272437)
	handleError (main.8e46cddcf277f36d.js:1:272585)
	next (main.8e46cddcf277f36d.js:1:353229)
	next
	next (main.8e46cddcf277f36d.js:1:143449)
	_next (main.8e46cddcf277f36d.js:1:143116)
	next (main.8e46cddcf277f36d.js:1:142803)
	(anonymous function) (main.8e46cddcf277f36d.js:1:140695)
	U (main.8e46cddcf277f36d.js:1:163913)
	next (main.8e46cddcf277f36d.js:1:140528)
	emit (main.8e46cddcf277f36d.js:1:333368)
	run (polyfills.813ff8f4ba583420.js:1:61615)
	onHandleError (main.8e46cddcf277f36d.js:1:347358)
	runGuarded (polyfills.813ff8f4ba583420.js:1:61806)
	(anonymous function) (polyfills.813ff8f4ba583420.js:1:74495)
	ie (polyfills.813ff8f4ba583420.js:1:69150)
	invokeTask (polyfills.813ff8f4ba583420.js:1:68089)
	Ee (polyfills.813ff8f4ba583420.js:1:80881)
	ve (polyfills.813ff8f4ba583420.js:1:81320)
```

Don't know why other API's are being blocked which are not even part of supertokens. Let me know if any other information is needed.

**Comment by rishabhpoddar** [comment]
Hey! This issue has been fixed in supertokens-website version >= 14.0.2.

Please clear node modules, and delete package-lock.json file and reinstall. Make sure that your package-lock.json says that 14.0.2 of supertokens-website is being used.

Closing this issue for now. But feel free to reopen in case the error persists.

**Comment by Alengeorge01** [comment]
Sure, Thanks a lot, I will try this out and let know if the error persists.
